Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -5890,3 +5890,9 @@ struct WebCore::RemoteMouseEventData {
     WebCore::FrameIdentifier targetFrameID;
     WebCore::IntPoint transformedPoint;
 };
+
+header: <WebCore/LinkDecorationFilteringData.h>
```

Comment:
Is `header: ` really necessary, since the struct and header name are the same?

Reply:
No, removed. I'll also remove the unnecessary for the struct above.